### PR TITLE
Make sure availableTo is never lower than availableFrom

### PR DESCRIPTION
**Pull request by bertramakers**
### Fixed

- Fixed some (test) events and places not being indexed because their `availableFrom` was higher than `availableTo`. There's no real use case for this, but hinders testing from external developers. To fix it we set `availableTo` to the `availableFrom` date if it's lower.

---

Ticket: https://jira.uitdatabank.be/browse/III-2877

### CODE REVIEW

**Comment by bertramakers** [comment]
All changes should be compatible with the silex EOL branch, so after merging we should be able to cherry-pick the commits to that branch.

**Comment by bertramakers** [comment]
I've also moved some code around to better structure it as part of an ongoing refactor of this part of the codebase. For easier reviewing, review the commits individually.

File: tests/ElasticSearch/Event/EventJsonDocumentTransformerTest.php [hunk 1]
```diff
@@ -462,6 +462,30 @@ public function it_uses_endDate_if_availableTo_is_missing()
         $this->assertEquals($expectedLogs, $actualLogs);
     }
 
+    /**
+     * @test
+     */
+    public function it_uses_makes_sure_availableFrom_is_never_higher_than_availableTo(): void
```
Comment (brampauwelyn) [inline tests/ElasticSearch/Event/EventJsonDocumentTransformerTest.php]: maybe a bit too long? 
why it_uses_makes_sure... and not just it_makes_sure... ?

Comment (bertramakers) [reply tests/ElasticSearch/Event/EventJsonDocumentTransformerTest.php]: Yep good point, I copy-pasted the test before it and didn't change the name correctly. Should be fixed now!

**Pull request merged**